### DISCUSSION TITLE
3d print file for a R3D Propane Gauge Module

### DISCUSSION THREAD

**Discussion by DavePanetta** [jgyates/genmon#1102] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

Genmon
Version V1.19.02

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

I was wondering if anyone had 3D printed the R3D Propane Gauge plastic tabs rather than buy a whole gauge just for the plastic tab to be used for a DIY tank reader. I don't have the skill or 3D printer to do it myself.

If I can get a file for one, I know someone that could print me a few.

Thanks
Dave

**Comment by jgyates** [comment]
I don't recall anyone doing this.
